src/app/components/menu/leftNav.jsx

```diff
@@ -1,13 +1,16 @@
 import React from "react";
 import Logo from "../../../shared/Logo";
 import logoImg from "../../../assets/images/logo.jpg";
+import { Link } from "react-router-dom";
 
 function LeftNav () {
+  
     return (
         <div className="images">
-        <Logo imageSrc={logoImg}/>
+        <Link to="/">
```

Comment:
Nadal masz złe formatownie.

Reply:
na moim komputerze to inaczej wyglada... moze przy pushowaniu cos robi blad?